iphone/package.json

```diff
@@ -20,7 +20,7 @@
 	"minWatchosVersion": "2.0",
 	"vendorDependencies": {
 		"xcode": ">=9.0 <=11.x",
```

Comment:
@ewanharris  Can you revert minimum Xcode version to 8.0 as well.

Reply:
@vijaysingh-axway, dang I missed that 😅 Just pushed it now